src/Features/Blockcore.Features.Wallet/Api/Controllers/WalletController.cs

```diff
@@ -1602,5 +1608,17 @@ private void SyncFromBestHeightForRecoveredWallets(DateTime walletCreationDate)
                 this.walletSyncManager.SyncFromHeight(blockHeightToSyncFrom);
             }
         }
+
+        private (byte[], bool) TryGetHexValue(string hexString)
```

Comment:
This is probably not the place for this method, don't we have an utility somewhere for this?

Reply:
@sondreb where should we place it? We have it there, as there was another method like this [SyncFromBestHeightForRecoveredWallets]

Reply:
I had a look at the method now and it just redirect the call to another utility library, so I guess it's fine =)